server/server.go

```diff
@@ -110,6 +111,10 @@ type WebConfig struct {
 	//
 	Dir string
 
+	// A file system includes web static. Will be overwritten by Dir
```

Comment:
The current wording is somewhat confusing. How about something like this:

"Alternative way to configure web static filesystem. Dir overrides this. It's expected to contain the same files and directories as mentioned above in Dir doc."